modules/@apostrophecms/login/index.js

```diff
@@ -682,6 +682,10 @@ module.exports = {
 
       // Awaitable wrapper for req.login. An implementation detail of the login route
       async passportLogin(req, user) {
+        const cookieName = `${self.apos.shortName}.${loggedInCookieName}`;
+        if (req.cookies[cookieName] !== 'true') {
+          req.res.cookie(cookieName, 'true');
+        }
```

Comment:
@boutell this is a quick fix, should we open an issue to investigate further this?